ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Use this template for reporting bugs.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
+
+### Bug Report
+
+#### Describe how to reproduce the bug
+
+Please provide a detailled description of what you have to do to reproduce your bug (settings, click paths, ...)
```

Comment:
detailed

Reply:
passive voice is easier to understand when describing generic behaviour: "Please provide a detailed description of what `has to be done` to reproduce the bug (settings, click paths, ...)